Aspire BudgetingUITests/Aspire_BudgetingUITests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Aspire_BudgetingUITests.swift
+//  Aspire BudgetingUITests
+//
+//  Created by TeraMo Labs on 3/7/20.
+//  Copyright © 2020 TeraMo Labs. All rights reserved.
+//
+
+import XCTest
+
+class Aspire_BudgetingUITests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/cab52a1edc5acb25f3d04b465494135b868d8155/Aspire BudgetingUITests/Aspire_BudgetingUITests.swift#L11"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>